heron/api/src/java/org/apache/heron/streamlet/impl/operators/TransformOperator.java

```diff
@@ -41,10 +40,10 @@
  */
 public class TransformOperator<R, T> extends StreamletOperator<R, T>
     implements IStatefulComponent<Serializable, Serializable> {
+
   private static final long serialVersionUID = 429297144878185182L;
   private SerializableTransformer<? super R, ? extends T> serializableTransformer;
 
-  private OutputCollector collector;
```

Comment:
I am thinking in the prepare() function, instead of setting collector direction, it might be better to call super.prepare(...) instead. What do you think?

Reply:
@nwangtw yep, i agree and submitted new patch